Godeps/_workspace/src/github.com/Azure/go-autorest/autorest/azure/token.go

```diff
@@ -1,12 +1,19 @@
 package azure
 
 import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha1"
+	"encoding/base64"
+	"fmt"
 	"net/http"
 	"net/url"
 	"strconv"
 	"time"
 
 	"github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/golang.org/x/crypto/pkcs12"
+	"github.com/dgrijalva/jwt-go"
```

Comment:
That was going to be my remark.

We don't seem to be vendoring sub-dependencies.

This is already the case with another dependency: `gopkg.in/check.v1`

I can introduce another commit to vendor both `jwt-go` and `gopkg.in/check.v1` and rewrite paths... if that's what we want to do.